Rename variable name to uppercase NAME

radoondas opened the pull request:
This is just a minor naming update of the script.
Is there any reason for variable 'name' to be lowercase?

radoondas committed:
Rename variable name to uppercase NAME

This is just a minor naming update of the script.
In SHELL script usually all variables are uppercase. Is there any reason for variable name' to be lowercase?

damm commented:
is there any reason to have name in uppercase either? assignment of variables is case sensitive in shells as well (typically).

damm commented on load.sh:
```diff
@@ -87,37 +87,37 @@ echo "Loading dashboards to $ELASTICSEARCH in $KIBANA_INDEX"
-    $CURL -XPUT $ELASTICSEARCH/$KIBANA_INDEX/search/$name \
+    NAME=`basename $file .json`
+    echo "Loading search $NAME:"
+    $CURL -XPUT $ELASTICSEARCH/$KIBANA_INDEX/search/$NAME \
```
We could be very careful here and make it ${NAME}

damm commented on load.sh:
```diff
@@ -87,37 +87,37 @@ echo "Loading dashboards to $ELASTICSEARCH in $KIBANA_INDEX"
-    echo "Loading search $name:"
-    $CURL -XPUT $ELASTICSEARCH/$KIBANA_INDEX/search/$name \
+    NAME=`basename $file .json`
+    echo "Loading search $NAME:"
```
${NAME} as well

ruflin replied:
+1 on using ${NAME} in general. Just do be save.

radoondas commented:
Well yes. On the other hand I would suggest not only ${NAME}, but more curly braces where variables are in the strings to be more safe and consistenst within the script. I will create PR later, with additions.

ruflin commented:
@radoondas Thanks

radoondas committed:
Fix issue in elastic/beats-dashboards #69 and adds 'curly braces' in to the script

This is more as a draft to be discussed. it will work for any beat if you configure it's mask in configfile.
Script doesn't touch original files, it creates tmp file instead.
Please, have a look and comment.

radoondas committed:
Revert "Fix issue in elastic/beats-dashboards #69 and adds 'curly braces' in to the script"

This reverts commit 38f60b08d7ad220f70be133931e90ae1fc02b79c.

radoondas committed:
Merge remote-tracking branch 'upstream/master'

radoondas committed:
Update script with curly braces in the script in order to make it safer

radoondas committed:
Fix changed function call

radoondas committed:
Enable to rename hardcoded index names in visualizations while importing to ES

This modification expect config file, where necessary renaming is defined.
This file can be empty or doesn't need to exist.
Script doesn't touch original files, it creates temp files instead.
This also works with other custom visualizations - just add your names and your set to go.
This solution requires further discussion.

radoondas commented:
For the reference, I have prepared another commit to  my fork with curly braces in the script: 6e1948d

I have no PR so far. I believe, further discussion is required.

radoondas commented:
I believe, this one can be just closed. It is obsolete since #86 is now merged and #85 is in discussion.

ruflin commented:
Agree in closing this as changes are in #86 and #85
